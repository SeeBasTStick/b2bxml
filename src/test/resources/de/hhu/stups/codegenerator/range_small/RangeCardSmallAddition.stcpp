int main() {
    RangeCardSmall range;
    range.simulate();
    cout << range.getResult();
    return 0;
}