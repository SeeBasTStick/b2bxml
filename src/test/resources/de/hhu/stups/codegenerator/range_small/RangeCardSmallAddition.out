public static void main(String[] args) {
    RangeCardSmall range = new RangeCardSmall();
    range.simulate();
    System.out.println(range.result);
}